targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -124,22 +124,27 @@ object CoordinatesForm {
             focus = true,
             loading = state.searching,
             error = state.searchTerm.isEmpty,
+            onTextChange = (u: String) =>
+              $.setStateL(State.searchTerm)(u) *> $.setStateL(State.searchError)(none),
             onBlur = (u: ValidatedNec[String, String]) =>
               u.toOption
                 .flatMap(
                   refineV[NonEmpty](_).toOption
                     .map(props.onNameChange(_))
                 )
                 .getOrEmpty,
-            onValidChange = (_: Boolean) => $.setStateL(State.searchError)(none),
+            disabled = state.searching,
             icon = Icons.Search
               .link(true)
```

Comment:
It seems like the `action` in `FormInputEV` is meant for this sort of thing. But, I'm not sure how to use it because the examples don't actually do anything when it is clicked/enter is hit.

Reply:
An unrealated comment regarding this line: it seems `State.searchError` is not being set anywhere now. Maybe we can change it to `searchEnabled: Boolean = true` and have here `onValidChange = v => $.setStateL(State.searchEnabled)(v)`.

Reply:
`searchError` is set in response to a failure querying `aladin` if you look above.

Reply:
@toddburnside I'll look at `action` it may help to a bug I'm seeing and it is nicer than the hidden submit field

Reply:
`action` doesn't help much here as it is supposed to contain the url you are submitting too.

Reply:
Ah. Interesting.